src/web/components/Icon/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Icon({ children, color, size }) {
+  const containerclasses = `icon ${color ? `has-text-${color}` : ''} ${size ? `is-${size}` : ''}`;
+
```

Comment:
I think you can do something like:

```javascript
const containerclasses = `icon ${color && `has-text-${color}`} ${size && `is-${size}`}`;
```

Reply:
Oh great, I forgot about that syntax. thanks